schema/schema_v1.json

```diff
@@ -0,0 +1,675 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "eq_id": {
+    "type": "string",
+    "description": "Used in combination with the form_type to uniquely identify a questionnaire.",
+    "required": "false"
+  },
+  "form_type": {
+    "type": "string",
+    "description": "Used in combination with the eq_id to uniquely identify a questionnaire.",
+    "required": "false"
+  },
```

Comment:
`eq_id` and `form_type` were moved in the eq-survey-runner pull request https://github.com/ONSdigital/eq-survey-runner/pull/1298. Do we have a story to remove the schemas from survey runner and run this as a service in the build?